test/unit_tests/modules/util/behavior-spec.js

```diff
@@ -0,0 +1,60 @@
+'use strict';
+
+var BASE_JS_PATH = '../../../../cfgov/unprocessed/js/';
+
+var chai = require( 'chai' );
+var expect = chai.expect;
+var jsdom = require( 'jsdom' );
+
+var behavior = require( BASE_JS_PATH + 'modules/util/behavior' );
+
+describe( 'behavior', function() {
+
+  var HTML_SNIPPET = '<div data-js-hook="behavior_flyout-menu">' +
+                     '<div data-js-hook="behavior_flyout-menu_content">' +
+                     '</div></div>';
+  var initdom = jsdom.jsdom( HTML_SNIPPET );
+  var document = initdom.defaultView.document;
+
+  var containerDom;
+  var behaviorElmDom;
+  var selector = '[data-js-hook=behavior_flyout-menu';
```

Comment:
Makes sense, updated in https://github.com/cfpb/cfgov-refresh/pull/2142/commits/e1d6e961302bcf3879de10222636289663a053c8